Source/DafnyCore/Rewriters/PrecedenceLinter.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Dafny {
   public class PrecedenceLinter : IRewriter {
     private readonly CompilationData compilation;
     // Don't perform linting on doo files in general, since the source has already been processed.
-    internal override void PostResolve(ModuleDefinition moduleDefinition) {
+    internal override void PreResolve(ModuleDefinition moduleDefinition) {
```

Comment:
Is the preceding comment still correct after this change?

Reply:
I don't know. What do you think?